src/components/Trends/TrendsPanel.js

```diff
@@ -26,7 +26,7 @@ import Select from '../RefineBar/Select';
 import { Separator } from '../RefineBar/Separator';
 import StackedAreaChart from '../Charts/StackedAreaChart';
 import { TabbedNavigation } from '../TabbedNavigation';
-import TrendDepthToggle from './TrendDepthToggle';
+import { TrendDepthToggle } from './TrendDepthToggle';
```

Comment:
Is there a standard around how to export these components? I've been doing `export default`, which causes it to import like the removed version of line 29, but also noticed it being exported as a variable.

Reply:
I prefer named exports with the brackets and I think we should stick to one convention.  Looks like there's also benefits to being explicit with the named exports.


https://react.dev/learn/importing-and-exporting-components#default-vs-named-exports
https://dev.to/phuocng/avoid-using-default-exports-a1c#:~:text=Default%20exports%20are%20used%20to,a%20function%20or%20a%20class.

We should see if we can find an eslint rule to enforce this standard.

Reply:
Also with the named imports, you can have more than one export per file and you can unit test them more easily.